modules/rosa-adding-node-labels.adoc

```diff
@@ -10,6 +10,11 @@ Add or edit labels for worker nodes at any time to manage the nodes in a manner
 
 Labels are assigned as key=value pairs. Each key must be unique to the object it is assigned to. Labels do not change or impact the core system values, such as a machine pool ID.
 
+[NOTE]
+====
+Currently, adding node labels on an existing machine pool will only add the labels to new nodes that are created in that pool.  The existing nodes will not have the node label applied.  If you need nodes with the node label, you can either scale down the nodes in the machine pool to zero (this will not work with the 'default' machine pool), add the node label, and then, scale back up to the desired amount.  You can also create a new machine pool and define the node labels at creation time.
```

Comment:
s/will only add/adds

You could also move the only closer to _new nodes_: _adds labels to only new nodes_ or _adds labels to new nodes only_.

Reply:
s/and then, / and then ?? (Do you need the comma there?)